site/src/patterns/hero.md

```diff
@@ -50,3 +50,32 @@ Hero with a theme-aware companion image.
 	</div>
 </section>
 ```
+
+## Accent hero with right content
```

Comment:
nit: In the paragraph below, this is referred to as the accent hero with details - we should make the heading consistent:

```suggestion
## Accent hero with details
```